### PR TITLE
Added RetroPage2 component with some CSS and  tested with App.js

### PR DESCRIPTION
**Pull request by SpartanDeBoss**
# Description
* Retro page 2
Please provide a short summary explaining what this PR is about.
* Some functionality to join a retro or to start a new retro
Fixes # (issue)

# How to test?
* yarn start
Please provide a short summary how your changes can be tested?
* It can be tested yet still have some functionality to add and make it work.
# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by shpomp** [comment]
@SpartanDeBoss  fix the build fail. 
See the error, it is related to Prettier and check scripts in package.json to see what to run 💡

**Comment by SpartanDeBoss** [comment]
> @SpartanDeBoss fix the build fail. See the error, it is related to Prettier and check scripts in package.json to see what to run 💡

it seems my prettier was not functioning so i disabled and enabled it after (formatting) the magic worked Thanks! @shpomp

File: packages/client/src/containers/RetroPageSecond/RetroPage2.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+          <div className="question">
+            {question.text}
+            <div className="box">
+              {comments[question.id] &&
+                comments[question.id].map((comment) => (
+                  <div key={comment.id} className="comment">
+                    {comment.text}
+                  </div>
+                ))}
+              <select className="roleSelect">
```
Comment (DanJecu) [inline packages/client/src/containers/RetroPageSecond/RetroPage2.jsx:121]: This one is not working. Could you figure out why?

**Review by DanJecu — CHANGES_REQUESTED** [review]
Good job Aaron 💪🏻 

I left you some comments to resolve.

**Review by DanJecu — CHANGES_REQUESTED** [review]
Names of components and functions should provide clarity, enabling you to navigate the code. So, if your app is a Retro app, it seems like `RetroPage` should be the `HomePage`, and `RetroPage2` should actually be `RetroPage`.
You can also call `RetroPage` --> `JoinRetroPage` & `RetroPage2` coud be `RetroPage`.

**Comment by SpartanDeBoss** [comment]
I have resolved those changes.

**Review by DanJecu — APPROVED** [review]

**Pull request merged**